modules/diskq/tests/Makefile.am

```diff
@@ -5,6 +5,7 @@ DISKQ_TEST_LD_ADD = $(TEST_LDADD)
 modules_diskq_tests_TESTS = \
   modules/diskq/tests/test_diskq \
   modules/diskq/tests/test_diskq_full \
+  modules/diskq/tests/test_diskq_truncate \
```

Comment:
Please add the tests to CMakeLists.txt too.

Reply:
Thanks, I will fix it.